earthpy/plot.py

```diff
@@ -386,9 +385,10 @@ def hist(
 
 def make_col_list(unique_vals, nclasses=None, cmap=None):
     """
-    Take a defined matplotlib colormap, and create a list of colors based on
-    a set of values. This is useful when you need to plot a series of
-    classified numpy arrays that are missing some of the sequential classes.
+    Take a matplotlib colormap and create a list of colors.
```

Comment:
How about: 
`Convert a matplotlib named colormap into a discrete list of n-colors in RGB format.`

I think you can remove the rest of the text then !